package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-scm-base",
-  "version": "2.4.0",
+  "version": "3.0.0",
```

Comment:
I think determining whether this is a major version bump or not depends on _whether the package that is using this module will break after pulling this change_. In this case, `scm-github` and `scm-bitbucket` will break after pulling this version, that's why I made it a major bump.